src/main/java/com/gildedrose/Conjured.java

```diff
@@ -1,27 +1,36 @@
 package com.gildedrose;
 
 public class Conjured extends DefaultItem {
-    private int conjuredQualityDegradation;
+    private final int conjuredQualityDegradationPastSellIn;
+    private final int conjuredQualityDegradation;
+    private final Item item;
 
-    public Conjured() {
+    public Conjured(Item item) {
+        super(item);
+        this.item = item;
         this.conjuredQualityDegradation = super.qualityDegradation * 2;
+        this.conjuredQualityDegradationPastSellIn = this.conjuredQualityDegradation * 2;
     }
 
-    public void age(Item item) {
-        item.quality = updateQuality(item.quality, item.sellIn);
-        item.sellIn = updateSellIn(item.sellIn);
+    @Override
+    public void age() {
+        item.quality = updateQuality();
+        item.sellIn = updateSellIn();
     }
 
-    private int updateQuality(int quality, int sellIn) {
-        if (sellIn <= 0) {
-            conjuredQualityDegradation = conjuredQualityDegradation * 2;
+    private int updateQuality() {
+        if (item.sellIn <= 0) {
+            item.quality -= conjuredQualityDegradationPastSellIn;
         }
-        quality -= conjuredQualityDegradation;
-        return Math.max(quality, 0);
+        else {
+            item.quality -= conjuredQualityDegradation;
+        }
```

Comment:
Suggestion: Possible to reduce the amount of if... else... statements w/ ternaries or something else more concise?